crabcache/templates/deployment.yaml

```diff
@@ -0,0 +1,182 @@
+        - name: robot-secrets
+          mountPath: /etc/robots
+          readOnly: true
+        - name: hmac-secrets
+          mountPath: /etc/hmac
+          readOnly: true
+        - mountPath: /host/etc/grid-security
+          name: etc-grid-security
+          readOnly: true
+        - name: setup-certs-and-run
+          mountPath: /opt/setup-certs-and-run
+      {{- if or .Values.environment.prod .Values.environment.preprod }}
+        - name: crabcache-storage
+          mountPath: /data/srv/state/crabcache/files
+        - name: logs
+          mountPath: /data/srv/logs/crabcache
+      - name: {{ .Values.image2.name}}
+        image: {{  .Values.image2.path }}
+        args: [
+          "-c", "/etc/filebeat.yml",
+          "-e",
+         ]
+        env:
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        resources:
+        {{- toYaml .Values.image2resources | nindent 12 }}
+        volumeMounts:
+         - name: logs
+           mountPath: /data/srv/logs/crabcache
+         - name: config
+           mountPath: /etc/filebeat.yml
+           readOnly: true
+           subPath: filebeat.yml
+         - name: data
+           mountPath: /usr/share/filebeat/data
+         - name: varlog
+           mountPath: /var/log
+         - name: varlibdockercontainers
+           mountPath: /var/lib/docker/containers
+           readOnly: true
+       {{- end }}
+      - image: "{{ .Values.imageExp.path }}:{{ .Values.imageExp.tag}}"
+        name: {{ .Values.imageExp.name }}
+        imagePullPolicy: {{ .Values.imageExp.imagePullPolicy }}
+        command:
+        {{- toYaml .Values.imageExp.command | nindent 10 }}
+        livenessProbe:
+         {{- toYaml .Values.imageExpLivenessProbe | nindent 12 }}
+        readinessProbe:
+         {{- toYaml .Values.imageExpReadinessProbe | nindent 12 }}
+        args:
+         {{- toYaml .Values.imageExp.args | nindent 10 }}
+        resources:
+        {{- toYaml .Values.imageExpResources | nindent 12 }}
+        volumeMounts:
+        - name: hmac-secrets
+          mountPath: /etc/hmac
+          readOnly: true
+        securityContext:
+          privileged: true
+      volumes:
+      - name: proxy-secrets
+        secret:
+          secretName: proxy-secrets
+      - name: secrets
+        secret:
+          secretName: crabcache-secrets
+      - name: robot-secrets
+        secret:
+          secretName: robot-secrets
+      - name: hmac-secrets
+        secret:
+          secretName: hmac-secrets
+      - name: etc-grid-security
+        hostPath:
+          path: /etc/grid-security
+      - name: setup-certs-and-run
+        configMap:
+          name: crabcache
+{{- if or .Values.environment.prod .Values.environment.preprod }}
+      - name: crabcache-storage
+        persistentVolumeClaim:
+{{- if .Values.environment.preprod }}
+            claimName: logs-cephfs-claim-preprod-crab
```

Comment:
claimName is `crabcache-claim` for both preprod and prod for `crabcache-storage`